### ISSUE TITLE
Option for weekly summary of activities?

### ISSUE THREAD

**Issue by kdliu86** [dblock/slack-strava#110]
Could this feature be added?  For users to receive a digest instead of every activity.

**Comment by dblock** [comment]
It's a good idea. Care to develop how that would work? Do we roll up the activities and post once? Do users get new options of when to post?

**Comment by kdliu86** [comment]
Yeah perhaps an end of week summary of "X completed these activities" in a larger message instead of individual activity submissions; for a larger channel we have almost too many activities showing up which basically blocks out all other related chat.
